### PR TITLE
Order

### PR DESCRIPTION
**Pull request by Elliefye**

### CODE REVIEW

File: backend/JiperBackend/Controllers/UserController.cs [hunk 1]
```diff
@@ -61,5 +63,50 @@ public IActionResult Login([FromBody] JObject data)
             
             return Ok(user);
         }
+
+        [HttpPost("neworder")]
+        public IActionResult NewOrder([FromBody] JObject data, [FromServices] CourierCompanyService courierCompanyService)
```
Comment (rimgailez) [inline backend/JiperBackend/Controllers/UserController.cs]: Maybe rename controller method from "NewOrder" to "AddOrder" or "CreateOrder"?

File: backend/JiperBackend/Examples/NewOrder.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "paymentType": "CARD",
+    "userId": "7",
+    "senderAddress": {
+        "city": "city",
+        "street": "str",
+        "houseNr": "1354",
+        "zipCode": "6545"
+    },
+    "package": "1",
```
Comment (rimgailez) [inline backend/JiperBackend/Examples/NewOrder.json]: I would suggest naming it to "packageId".

Comment (Elliefye) [reply backend/JiperBackend/Examples/NewOrder.json]: Ignas said he named it "package" in frontend so I left it like that

File: backend/JiperBackend/Controllers/UserController.cs [hunk 3]
```diff
@@ -61,5 +63,50 @@ public IActionResult Login([FromBody] JObject data)
             
             return Ok(user);
         }
+
+        [HttpPost("addorder")]
```
Comment (rimgailez) [inline backend/JiperBackend/Controllers/UserController.cs]: You don't need to change the name of endpoint - I think that "newOrder" could have stayed. I just wanted for you to change the controller method name. :)

Comment (Elliefye) [reply backend/JiperBackend/Controllers/UserController.cs]: soooo change again?

Comment (rimgailez) [reply backend/JiperBackend/Controllers/UserController.cs]: I think so. :)

**Review by rimgailez — APPROVED** [review]

**Pull request merged**